app/reports/protocols.rb

```diff
@@ -85,8 +85,10 @@ def column_attrs
     attrs["Primary PI Last Name"] = "service_request.try(:protocol).try(:primary_principal_investigator).try(:last_name)"
     attrs["Primary PI First Name"] = "service_request.try(:protocol).try(:primary_principal_investigator).try(:first_name)"
     attrs["Primary PI Email"] = "service_request.try(:protocol).try(:primary_principal_investigator).try(:email)"
-    attrs["Primary PI College"] = ["service_request.try(:protocol).try(:primary_principal_investigator).try(:college)", COLLEGES.invert] # we invert since our hash is setup {"Bio Medical" => "bio_med"} for some crazy reason
```

Comment:
Should the constants COLLEGES and DEPARTMENTS be removed or are they still being used elsewhere in the application?